src/app/screens/ordinals/ordinalImage.tsx

```diff
@@ -237,7 +241,8 @@ function OrdinalImage({
     );
   }
 
-  if (contentType.includes('image')) {
+  // if content type is undefined or "", we treat it the same as an image and fall back to ordiview thumbnail
+  if (!contentType || contentType.includes('image')) {
```

Comment:
dunno what PR will be merged first but [this](https://github.com/secretkeylabs/xverse-web-extension/pull/777/commits/89a84308929e7bc72689b1671908c17f3ccd2b02) change maybe overwritten right?

Reply:
yup.. good catch